draft-ietf-scitt-architecture.md

```diff
@@ -240,7 +238,7 @@ and on prior Signed Statements already added to a Registry.
 Registry:
 
 : the verifiable append-only data structure that stores Signed Statements in a Transparency Service often referred to by the synonym log or ledger.
-SCITT supports multiple Registry and Receipt formats to accommodate different Transparency Service implementations, such as historical Merkle Trees and sparse Merkle Trees.
+Since COSE Signed Merkle Tree Proofs ({{-COSEMTP}}) support multiple Merkle Tree algorithms, SCITT supports different Transparency Service implementations of the Registry, such as historical Merkle Trees or sparse Merkle Trees.
```

Comment:
```suggestion
Since COSE Signed Merkle Tree Proofs ({{-COMETRE}}) support multiple Merkle Tree algorithms, SCITT supports different Transparency Service implementations of the Registry, such as historical Merkle Trees or sparse Merkle Trees.
```